packages/pigeon/lib/ast.dart

```diff
@@ -277,7 +277,7 @@ class Parameter extends NamedType {
     bool? isPositional,
     bool? isRequired,
     super.documentationComments,
-  })  : isNamed = isNamed ?? true,
```

Comment:
This fixes existing tests that don't specify this but still assume it's false.

Since this wasn't used anywhere besides toString, I don't consider this a breaking change.